_stories/molecules/SearchMultiSelect/index.js

```diff
@@ -81,6 +81,11 @@ const namesList = [
         name: 'David "Henry" Mejorado',
         isSelected: false,
         key: 4
+    },
+    {
+        name: 'Mahdiye / Maddie Jamali',
```

Comment:
one minor thing, this branch fixes the backslash `\`. the forward slash worked fine already!